netmanager/src/views/apis/deviceRegistry.js

```diff
@@ -136,26 +136,31 @@ export const getEventsApi = async (params) => {
 };
 
 export const getSitesApi = async (params) => {
+  axios.defaults.headers.common.Authorization = jwtToken;
```

Comment:
Hi @OchiengPaul442 , this is great! Thanks!

I am just curious, can't this HEADERS setting just be done once? And then using that configured `axios` across the entire file? I am sensing some duplication here.

Also, do we also send the TOKEN as a query parameter or it was removed as earlier requested.